templates/default/ckan_properties.ini.erb

```diff
@@ -6,14 +6,12 @@
 # following URL for a description of what they do and the full list of
 # available options:
 #
-# http://docs.ckan.org/en/ckan-2.2.1/configuration.html
+# http://docs.ckan.org/en/latest/maintaining/configuration.html
 #
 # The %(here)s variable will be replaced with the parent directory of this file
 #
 
 [DEFAULT]
-
-# WARNING: *THIS SETTING MUST BE SET TO FALSE ON A PRODUCTION ENVIRONMENT*
```

Comment:
why remove this comment?

Reply:
It's not needed here, and I wanted the file to be more similar to the ones used in testing.